controllers/homework.js

```diff
@@ -672,15 +678,18 @@ router.get('/:assignmentId/edit', (req, res, next) => {
 			return next(error);
 		}
 
-		assignment.availableDate = moment(assignment.availableDate).format('DD.MM.YYYY HH:mm');
-		assignment.dueDate = moment(assignment.dueDate).format('DD.MM.YYYY HH:mm');
+		assignment.availableDate = timesHelper.fromUTC(assignment.availableDate).format('DD.MM.YYYY HH:mm');
+		assignment.dueDate = timesHelper.fromUTC(assignment.dueDate).format('DD.MM.YYYY HH:mm');
 
 		addClearNameForFileIds(assignment);
 		// assignment.submissions = assignment.submissions.map((s) => { return { submission: s }; });
 
 		const coursesPromise = getSelectOptions(req, `users/${res.locals.currentUser._id}/courses`, {
 			$limit: false,
 		});
+
+		const timezone = timesHelper.getChangedTimezoneString();
```

Comment:
we should use global variables here https://ddcode.net/2019/04/20/how-does-express-handlebars-pass-global-variables-to-templates/